ttnn/cpp/ttnn/operations/data_movement.hpp

```diff
@@ -283,8 +284,40 @@ struct Repeat {
     }
 };
 
+struct RepeatInterleave {
+    static inline const std::array<TensorSchema, 1> input_tensor_schemas() {
+        return {ttnn::TensorSchema{
+            4,  // min rank
+            4,  // max rank
+            {ttnn::bfloat16},
```

Comment:
I'd like to learn what determines compatibility. 
E.g. why uint16 is not working too

Reply:
good question, that's really op implementation dependent and requires communicating with the op-writer to understand the spec of what is/isn't supported.